Traceback (most recent call last):
  File "main.py", line 125, in <module>
    managerial_layer()
  File "main.py", line 121, in managerial_layer
    unfold_network(cpc_time_series, tensors, topical_patents[:2]) 
  File "/cluster/raid/home/michael.tsesmelis/SideProject/network_analysis.py", line 147, in unfold_network
    assignee_df = assignee_index(topical_assignees, tensors["assignee"])
  File "/cluster/raid/home/michael.tsesmelis/SideProject/network_analysis.py", line 76, in assignee_index
    assignees_df['emergingness'] = assignees_df.apply(lambda row: sum(topical_assignees[row['ID']]['emergingness']) / row['count'])
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/frame.py", line 8740, in apply
    return op.apply()
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/apply.py", line 688, in apply
    return self.apply_standard()
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/apply.py", line 812, in apply_standard
    results, res_index = self.apply_series_generator()
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/apply.py", line 828, in apply_series_generator
    results[i] = self.f(v)
  File "/cluster/raid/home/michael.tsesmelis/SideProject/network_analysis.py", line 76, in <lambda>
    assignees_df['emergingness'] = assignees_df.apply(lambda row: sum(topical_assignees[row['ID']]['emergingness']) / row['count'])
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/series.py", line 942, in __getitem__
    return self._get_value(key)
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/series.py", line 1051, in _get_value
    loc = self.index.get_loc(label)
  File "/cluster/raid/home/michael.tsesmelis/SideProject/lib/python3.8/site-packages/pandas/core/indexes/range.py", line 388, in get_loc
    raise KeyError(key)
KeyError: 'ID'
